chan_series.go

```diff
@@ -159,13 +159,14 @@ func makeNodeAnn(n *channeldb.LightningNode) (*lnwire.NodeAnnouncement, error) {
 		return nil, err
 	}
 	return &lnwire.NodeAnnouncement{
-		Signature: wireSig,
-		Timestamp: uint32(n.LastUpdate.Unix()),
-		Addresses: n.Addresses,
-		NodeID:    n.PubKeyBytes,
-		Features:  n.Features.RawFeatureVector,
-		RGBColor:  n.Color,
-		Alias:     alias,
+		Signature:       wireSig,
+		Timestamp:       uint32(n.LastUpdate.Unix()),
+		Addresses:       n.Addresses,
+		NodeID:          n.PubKeyBytes,
+		Features:        n.Features.RawFeatureVector,
+		RGBColor:        n.Color,
+		Alias:           alias,
+		ExtraOpauqeData: e1.ExtraOpauqeData,
```

Comment:
Spelling is wrong: `ExtraOpauqeData` should be `ExtraOpaqueData` (here, and in all the places below)